roles/galaxy-route/defaults/main.yml

```diff
@@ -2,19 +2,20 @@
 # Host to create the root with.
 # If not specific will default to <instance-name>-<namespace>-<routerCanonicalHostname>
 #
-route_host: ''
 
+# common default values
```

Comment:
`nginx_read_timeout` could be moved here and used on the nginx vars below

Reply:
We're trying to make sure we use client_request_timeout for all of the possible locations so we have one similar source of "truth"